unlock-app/src/__tests__/services/web3Service.test.js

```diff
@@ -6,6 +6,7 @@ import nock from 'nock'
 import Web3Service from '../../services/web3Service'
 import UnlockContract from '../../artifacts/contracts/Unlock.json'
 import LockContract from '../../artifacts/contracts/PublicLock.json'
+import configure from '../../config'
```

Comment:
Since you're just using the `provider` property, maybe you can skip that and `configure(global)` and just create a fake `providers` object?

Reply:
I tried this, but we need other aspects of the provider internally for the tests to pass